channels/set.go

```diff
@@ -27,6 +27,15 @@ const UserStarChannel = "*"     // user channel for "can access all docs"
 const DocumentStarChannel = "!" // doc channel for "visible to all users"
 const AllChannelWildcard = "*"  // wildcard for 'all channels'
 
+type ID struct {
```

Comment:
Although it's in the channels package, I still wonder if ChannelID would be more self-documenting and clear.